packages/expo-updates/scripts/createManifest.js

```diff
@@ -65,6 +65,8 @@ const filterPlatformAssetScales = require('./filterPlatformAssetScales');
 
   fs.writeFileSync(path.join(destinationDir, 'app.manifest'), JSON.stringify(manifest));
 })().catch(e => {
+  // Wrap in regex to make it easier for log parsers (like `@expo/xcpretty`) to find this error.
+  e.message = `@build-script-error-begin\n${e.message}\n@build-script-error-end\n`
```

Comment:
in theory this could catch other, non-metro errors, as well. is that something the log formatter can handle? you could put this on line 27 instead, which is the only place we call metro.